rust/src/sip/sip.rs

```diff
@@ -173,17 +224,60 @@ impl SIPState {
             }
         }
     }
+
+    fn parse_response_tcp(&mut self, flow: *const core::Flow, stream_slice: StreamSlice) -> AppLayerResult {
+        let input = stream_slice.as_slice();
+        if input.is_empty() {
+            return AppLayerResult::ok();
+        }
+        let _pdu = Frame::new(
```

Comment:
I think this not a PDU, the PDU is the part of the stream slice corresponding to the `response` a few lines below